xonsh/cli_utils.py

```diff
@@ -105,10 +121,13 @@ def get_doc(func: tp.Union[tp.Callable, str], parameter: str = None):
 
     doc = inspect.getdoc(func) or ""
```

Comment:
So call `cleandoc` here before passing the docs to `_get_epilog_doc` and friends

Reply:
it already does https://docs.python.org/3/library/inspect.html#inspect.getdoc